CHANGES.rst

```diff
@@ -1,3 +1,10 @@
+v3.2.0
+======
+
+* #265: ``EntryPoint`` objects now expose a ``.dist`` object
+  referencing the ``Distribution`` when constructed from a
```

Comment:
When?
An entry point exists based on a distribution defining it right?
Or, in which case does an entrypoint exist without a distribution?
So, why when

Reply:
The test suite in particular constructs EntryPoints without a distribution. I wouldn't expect library consumers to construct EntryPoints this way, so I'm not describing it here, and instead focusing on the use-case that users would expect to see.